### ISSUE TITLE
Poll Expiration

### ISSUE THREAD

**Issue by lukeed** [apex/gh-polls#28]
I personally don't have a use case for this, but it might be nice to be able to set an expiration date/time for certain polls.

**Comment by tj** [comment]
Might help keep my dynamo clean haha, could become a wasteland of dead polls. Maybe some reasonable expiration by default like 6 months

**Comment by lukeed** [comment]
Right! Although, it might be nice to keep poll results for historical records. Perhaps convert to PNG at ~6 months & move to a static bucket?

I wasn't being clear though -- I meant that there'd be a time limit on casting polls, like Twitter. For example, create a new GH-Poll that will close in 7 days.

**Comment by tj** [comment]
Ah yea I guess it wouldn't be kind to just delete them :D haha.

**Comment by m4heshd** [comment]
@tj Is there any way to close a poll at the moment?

**Comment by tj** [comment]
@m4heshd ahh nope, I didn't anything for that
